### PR TITLE
Avoid hard-coding 1470

### PR DESCRIPTION
**Pull request by ArneBab**
this should work, though I had to install timeparser from PyPI to test it.

### CODE REVIEW

File: build_autoupgrade [hunk 1]
```diff
@@ -28,20 +28,19 @@ def main():
...
-          old_fetch, new_fetch, "1471-freenet.jar")
+          old_fetch, new_fetch, "{}-freenet.jar".format(transition_build))
 
     # Second build updated to; run by node serving updates.
     # Non-mandatory is for updating with the keys; mandatory is for UOM.
     for mandatory in [True, False]:
         build("java7-update-key", transition_build, continuation_build,
               mandatory, old_fetch, new_fetch,
-              "1471-{}mandatory-freenet.jar".format(
+              "{}-{}mandatory-freenet.jar".format(transition_build,
```
Comment (Thynix) [inline build_autoupgrade]: This was wrong before - it should be continuation_build. I made that change locally already and will persist it.

Comment (ArneBab) [inline build_autoupgrade]: do you want to change it from this pull-request?

**Comment by Thynix** [comment]
Thanks!

**Comment by ArneBab** [comment]
Thank you!

**Pull request merged**